src/components/SiteFooter/SiteFooter.tsx

```diff
@@ -30,19 +31,22 @@ const SiteFooter = ({ theme = FooterTheme.Light }: Props) => (
           : `3px solid ${colors.secondary.light5}`,
     }}
   >
-    <div className="flex justify-center items-center">
+    <Text className="flex justify-center items-center">
       © 2013–{DateTime.local().year} Jesse Stuart
-    </div>
-    <div className="dib center tc flex-ns justify-center items-center">
+    </Text>
+    <Text
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)